src/interface/validators.ts

```diff
@@ -13,4 +13,26 @@ export const hospitalRegRule = {
 export const hospitalLogInRule = {
     email: "required|email",
     password: "string|max:8|min:5",
+}
+
+export const patientRegRule = {
+    firstName: "string|required",
+    lastName: "string|required",
+    otherName: "string",
+    email: "required|email",
+    password: "string|max:8|min:5",
+    phoneNumber: "string|required|min:11",
+    genotype: "string",
+    bloodGroup: "string",
+    passport: "string",
+    nationality: "string",
+    state: "string",
+    localGovernmentArea: "string",
+    nin: "string",
+    dateOfBirth: "string"
```

Comment:
add |date to the dateOfBirth so that it ensures the date is of correct format. you can find more here about validatorjs https://www.npmjs.com/package/validatorjs